Update KEB docs (#2189)
kyma-project/control-plane: docs/kyma-environment-broker/03-11-swagger.md

## Use Virtual Service

> Open one of the following websites:
> 
>    https://kyma-env-broker.cp.dev.kyma.cloud.sap/
>    ```

Author: ```suggestion
Open Kyma Environment Broker APIs on [development env](https://kyma-env-broker.cp.dev.kyma.cloud.sap/) or on [stage env](https://kyma-env-broker.cp.stage.kyma.cloud.sap/).

```

>    ```

Author: ```suggestion
  
```

>    https://kyma-env-broker.cp.stage.kyma.cloud.sap/
>    ```

Author: ```suggestion
```

## Use Virtual Service

Open one of the following websites:

>    ```

Reviewer: ```suggestion
```
If the next suggestion is applied, this has to be removed.

>    https://kyma-env-broker.cp.dev.kyma.cloud.sap/
>    ```
>    ```
>    https://kyma-env-broker.cp.stage.kyma.cloud.sap/
>    ```

Reviewer: ```suggestion
- [`https://kyma-env-broker.cp.dev.kyma.cloud.sap/`](https://kyma-env-broker.cp.dev.kyma.cloud.sap/)
- [`https://kyma-env-broker.cp.stage.kyma.cloud.sap/`](https://kyma-env-broker.cp.stage.kyma.cloud.sap/)
```
There's no need to format these as full code blocks, these are just values that we list. We can, however, link them so you don't need to copy and paste them, and just click to open.
